backend/endpoints/file.py

```diff
@@ -24,35 +29,61 @@
         # room_id: 123456
 # }
 
+"""
+    A helper function to handle Multiple Uploaded image files in a DM
+    f
+"""
+
 def zipfiles(filenames):
+    """
+     Receives a list of  uploaded files
+     Returns:
+        An Archive file object
+    """
     zip_filename = "archive.zip"
 
     data = io.BytesIO()
     Zipfile = zipfile.ZipFile(data, "w")
 
     for fpath in filenames:
-        try:
-            fdir, fname = os.path.split(fpath)
-        except:
-            fpath = fname
+        fdir, fname = os.path.split(fpath)
+        print(fname,fdir)
         Zipfile.write(fpath, fname)
     Zipfile.close()
     resp = Response(data.getvalue(), media_type="image/png", headers={
         'Content-Disposition': f'attachment;filename={zip_filename}'
     })
     return resp
 
-async def read_item(img: str):
+
+"""
+A helper asyn function to handle single Uploaded image file in a DM
+An endpoint decorator to put correct media type in Open API specification
+"""
+
+
+async def get_image(img: str):
+
     if img:
         return FileResponse("image.png", media_type="image/png")
     else:
         return {"file": "No file uploaded"}
 
 
+
+""""
+An Endpoint that handles viwing image files uploaded in the DM
+
+params:
+    org_id = Organisation id of user
+    room_id = room id of user
+
+Retrieves All Messages in the Dm with the get_room_message method
+Handles  conditons for single and Multiple Uploaded Files
+"""
 @router.get("/get/files/{org_id}/{room_id}")
```

Comment:
this url path is wrongly defined, check the codebase and observe the url naming patterns